changelog.md

```diff
@@ -2,17 +2,20 @@
 
 ## [Unreleased]
 
+## [0.2.0] - 2023-12-20
+
 ### Added
 
 - Query transactions now run in read-only mode, and allow transaction isolation level configuration. ([#209](https://github.com/hasura/ndc-postgres/pull/209), [#212](https://github.com/hasura/ndc-postgres/pull/212))
 - Support variables as arguments to native queries ([#211](https://github.com/hasura/ndc-postgres/pull/211))
 - Introduce version 2 of connector deployment configuration. ([#208](https://github.com/hasura/ndc-postgres/pull/208))
-- Support array types ([#191](https://github.com/hasura/ndc-postgres/pull/191), ...)
+- Support array types ([#191](https://github.com/hasura/ndc-postgres/pull/191))
 - Support Native Query Mutations ([#189](https://github.com/hasura/ndc-postgres/pull/189), [#198](https://github.com/hasura/ndc-postgres/pull/198))
 
 ## [0.1.0] - 2023-11-29
 
 Initial release.
 
-[Unreleased]: https://github.com/hasura/ndc-postgres/compare/v0.1.0...HEAD
+[Unreleased]: https://github.com/hasura/ndc-postgres/compare/v0.2.0...HEAD
```

Comment:
Hm.. why is this version list at the bottom of the file? It looks like this PR is making an addition to v0.1.0.

Wouldn't it be better to have the list at the top of the file? Sort of like a table of contents?

Reply:
This is not a list. This is markdown syntax for links that appear somewhere in the document (so you can write `[Unreleased]` and it refers to a link to this link.
Render this changelog and see :) https://github.com/hasura/ndc-postgres/blob/24010c16f5a9b41655989c9d2dde097b9fad91b7/changelog.md

Reply:
Ah, it seems like a simplified version of the footnote syntax. 

How neat - I didn't know about that.

Let's keep that then!